bugheist/settings.py

```diff
@@ -272,7 +272,7 @@
 SECURE_PROXY_SSL_HEADER = ("HTTP_X_FORWARDED_PROTO", "https")
 
 # Allow all host headers
-ALLOWED_HOSTS = [".bugheist.com", "127.0.0.1"]
+ALLOWED_HOSTS = ["*"]
```

Comment:
Please add this back, it makes it more secure

Reply:
I have done with this @fredfalcon .